api/tests/unit/domain/services/mail-service_test.js

```diff
@@ -271,31 +300,53 @@ describe('Unit | Service | MailService', () => {
       });
     });
 
-    context('When tags property is provided', () => {
+    context('When locale is fr-fr', () => {
```

Comment:
**suggestion**
La structure actuelle est 
When locale is X
> should call sendEmail with expected options
When locale is Y
> should call sendEmail with expected options
When locale is Z

Je propose:

Should call sendEmail with localized values but for tags
> when locale is X
> when locale is Y
> when locale is Z

> should call sendEmail with expected options
```suggestion
    context('When locale is fr-fr', () => {
```

Reply:
Je ne pense pas que l'on puisse inverser l'ordre des `should`/`when` comme tu le proposes car on à des `context` qui incluent des `it` et dans la majorité des tests, les contextes commencent par des when et les it par des shoulds.

Reply:
On garde ça, je ferais une BSR pour proposer mieux avec test parmétré